crs/v1alpha2/controlplane_centos.yaml

```diff
@@ -54,6 +54,7 @@ spec:
     - mkdir -p /home/centos/.kube
     - cp /etc/kubernetes/admin.conf /home/centos/.kube/config
     - chown centos:centos /home/centos/.kube/config
+    - KUBECONFIG=/etc/kubernetes/admin.conf kubectl apply -f https://docs.projectcalico.org/v3.11/manifests/calico.yaml
```

Comment:
would this work if kubeadm init was run with a different pod network ?
It seems to assume the pod network as shown here https://docs.projectcalico.org/getting-started/kubernetes/quickstart.

Would it be better to download the file and do replacement ?

Reply:
According to the manifest:
```
# The default IPv4 pool to create on startup if none exists. Pod IPs will be
# chosen from this range. Changing this value after installation will have
# no effect. This should fall within `--cluster-cidr`.
```
So I'm assuming as the CIDR is set here https://github.com/metal3-io/metal3-dev-env/blob/master/crs/v1alpha2/cluster.yaml that would be used.

Reply:
I think this might cause a problem. It needs to fall within `--cluster-cidr`, but cluster CIDR is 192.168.0.0/18 and calico default pool is 192.168.0.0/16, so by default it does not fall within. So I don't think it would work as is. the sed step should be probably be applied